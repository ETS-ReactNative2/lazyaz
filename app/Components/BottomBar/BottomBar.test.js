import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import { BottomBar } from './index';

test('BottomBar component renders correctly', () => {
  const props = {
    destination: {},
  };
  const tree = renderer.create(<BottomBar {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
