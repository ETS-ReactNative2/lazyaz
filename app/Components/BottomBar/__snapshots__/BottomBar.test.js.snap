// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomBar component renders correctly 1`] = `
<View>
  <Text />
  <View>
    <Image
      source={
        Object {
          "testUri": "../../../app/Images/Icons/sets.png",
        }
      }
    />
    <Text>
      0
    </Text>
    <Text>
      sets
    </Text>
  </View>
  <Text />
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          undefined,
          Object {
            "flexDirection": null,
            "height": 48,
            "width": 375,
          },
          undefined,
        ]
      }
    >
      <Text
        style={
          Array [
            Array [
              undefined,
              Object {
                "lineHeight": 48,
              },
            ],
            Object {
              "color": undefined,
            },
          ]
        }
      >
        VIEW
      </Text>
    </View>
  </View>
</View>
`;
